backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -101,14 +104,17 @@ public static void start(int port) {
         (req, res) -> {
           if (!("/" + OIDC_LOGIN_PATH).equals(req.pathInfo())
               && !("/" + OIDC_CALLBACK_PATH).equals(req.pathInfo())
+              && !("/:schema/sitemap.xml").equals(req.pathInfo())
               && !("/" + ROBOTS_TXT).equals(req.pathInfo())) {
             res.redirect("/" + req.params(SCHEMA) + "/");
           }
         });
     before(
         "/:schema/:app",
         (req, res) -> {
-          if (!req.params("app").equals("graphql") && !req.params("app").equals("theme.css")) {
+          if (!req.params("app").equals("graphql")
+              && !req.params("app").equals("theme.css")
+              && !req.params("app").equals("sitemap.xml")) {
```

Comment:
I would prefer that you move these to the 'api' prefix. Instead of escaping everywhere. The graphql and theme.css are exceptions on that rule for good reasons.

Reply:
i would prefer to remove the before handlers that change the path these are to greedy

Reply:
they might not be necessary if we initialise handlers in correct order.